tests/antares/services/api_services/test_study_api.py

```diff
@@ -197,3 +197,6 @@ def test_create_binding_constraint_fails(self):
                 match=f"Could not create the binding constraint {constraint_name}: {self.antares_web_description_msg}",
             ):
                 self.study.create_binding_constraint(name=constraint_name)
+
+    def test_read_areas(self):
+        pass
```

Comment:
Indeed you should add tests here but you could also add one inside the test_web_client :)